dojo/unittests/test_spotbugs_parser.py

```diff
@@ -0,0 +1,16 @@
+from django.test import TestCase
+from dojo.tools.spotbugs.parser import SpotbugsXMLParser
+from dojo.models import Test
+
+
+class TestSpotbugsXMLParser(TestCase):
+
+    def test_no_findings(self):
+        parser = SpotbugsXMLParser(None, Test())
+        self.assertEqual(0, len(parser.items))
+
+    def test_parse_many_finding(self):
+        testfile = open("dojo/unittests/scans/spotbugs/many_findings.xml")
+        parser = SpotbugsXMLParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(81, len(parser.items))
```

Comment:
Could you add more unit tests on the attributes because you added more data to findings attributes.
Like that:
```Python
        # finding 4
        finding = findings[4]
        self.assertEqual("Hidden RPC Services", finding.title)
        self.assertEqual("Low", finding.severity)
        self.assertEqual(datetime(2019, 4, 2, 10, 14, 53, tzinfo=pytz.utc), finding.date)
        self.assertEqual("Some impact\n\n", finding.impact)
```